README.md

```diff
@@ -46,6 +46,7 @@ Each of these failures contribute to the number of retests that occur in CI and
 ![sig-storage-retests](https://kubevirt.io/ci-health/output/kubevirt/kubevirt/sig-storage-retests.svg)
 ![sig-network-retests](https://kubevirt.io/ci-health/output/kubevirt/kubevirt/sig-network-retests.svg)
 ![sig-operator-retests](https://kubevirt.io/ci-health/output/kubevirt/kubevirt/sig-operator-retests.svg)
+![sig-ci-retests](https://kubevirt.io/ci-health/output/kubevirt/kubevirt/sig-ci-retests.svg)
```

Comment:
why is it called sig-ci**-retests**? shouldn't it be failures? Or better another more descriptive thing, like sig-ci-infra-failures?

Reply:
This seems to follow through the whole PR...

Reply:
this is the number of CI failures that contributed directly to retests so thats where the name comes from.